docs/pages/_app.js

```diff
@@ -174,7 +174,7 @@ function AppWrapper(props) {
           },
           {
             text: 'View all versions',
-            href: `https://mui.com${languagePrefix}/versions/`,
+            href: `https://mui.com${languagePrefix}/material-ui/getting-started/versions/`,
```

Comment:
I'm not fully sure why this is using the hard-coded domain, but it will also break because it doesn't exist yet. If we leave it like this and merge this PR on `next`, it will break until it's hit `master`. Any tips/thoughts?

Reply:
IIUC, if we use the current domain (`next.mui.com`), it should be fine because these changes will only be on `next`. But I don't know if there's a valid reason to use the hardcoded domain.

Reply:
We'd need to change from `next.` to just `mui` once `master` returns to become the default branch, so unsure that's the right move. Intuitively, I'd have this without the domain, but I'm also not sure 🤷

Reply:
I mean using whatever variable there's available that points to the current domain.